project-metrics/metrics_service/database/models.py

```diff
@@ -66,20 +66,25 @@ class TravisState(enum.Enum):
   Based on
   https://github.com/travis-ci/travis-api/blob/master/lib/travis/model/job/test.rb#L21
```

Comment:
Still true?

Reply:
Updated to the `build` states instead of  `job` states.

Reply:
(in workspace, waiting to commit until other PR fixes are taken care of)